### ISSUE TITLE
Arrow next to Z Thermal Adjust, indicating ascending vs descending temp.

### ISSUE THREAD

**Issue by PhilBaz** [mainsail-crew/mainsail#2098]
### Requested feature:

Simple idea. An arrow next to the Z Thermal Adjust (in the temp panel) that would indicate if the temp is moving up or down.

The problem is that when the printer is (re)heating, after a previous print, it can be difficult to know whether the Z Thermal Adjust is actually moving down or up, since it takes a long time for this to turn around. It would be great to have an arrow to show the direction of travel. 

![Screenshot 2024-12-26 222241](https://github.com/user-attachments/assets/7dd31892-4c81-4ca1-857c-38e5eb87dc40)

### Solves the following problem:

Don't know if the frame temp is actually ascending or descending.

### Additional information:

Hope this makes sense... The basic theory is that bed meshing is more reliable when frame temp is moving up, rather than changing directions after the bed mesh is taken. 

Let me know if more info would be helpful.

**Comment by meteyou** [comment]
hey @PhilBaz! what should the algorithm for the symbol look like? because Moonraker sends the data at 200ms intervals, it would mostly display "the same" and then only briefly up or down. therefore, a more "advanced algorithm" would make more sense here.

**Comment by PhilBaz** [comment]
If possible maybe average of 30 or 40 seconds of Moonraker temp, then compare that against the next 30-40 seconds. If posible maybe a rolling average of  n-40 to n-80 seconds, subtracted from n-39 to n-0. if that makes sense?

I think part of what you are saying is that the symbol would only show for some amount of time when the temp had not stabilized. Like it would be there when rising or falling but not when relatively stable? Would it possibly make sense to just always have the last direction indicated even if it has stabilized?

Thanks for considering the request :)

**Comment by meteyou** [comment]
> I think part of what you are saying is that the symbol would only show for some amount of time when the temp had not stabilized. Like it would be there when rising or falling but not when relatively stable? Would it possibly make sense to just always have the last direction indicated even if it has stabilized?

@PhilBaz no, i didn't mean to say that. i just wanted to ask what the relevant range for calculating the direction should be. if i only query to the last data point, it would probably also show 90% of the time == and not up or down, because the temperature change is much too slow.

**Comment by PhilBaz** [comment]
@meteyou I see this is going stale... Didn't know if you would like me to leave it be. Did the approach above make any sense in terms of getting a useful symbol?
